src/Aspire.Hosting.Azure.Redis/AzureRedisCacheResource.cs

```diff
@@ -29,7 +29,7 @@ public class AzureRedisCacheResource(string name, Action<AzureResourceInfrastruc
     ///
```

Comment:
The XML docs above should be updated since this is no longer a `secret output reference`